src/Symfony/Component/Validator/Constraints/AllValidator.php

```diff
@@ -44,12 +44,12 @@ public function validate($value, Constraint $constraint)
             $validator = $context->getValidator()->inContext($context);
 
             foreach ($value as $key => $element) {
-                $validator->atPath('['.$key.']')->validate($element, $constraint->constraints);
+                $validator->atPath('['.$key.']')->validate($element, $constraint->constraints, array($context->getGroup()));
```

Comment:
Only subsets are allowed on traversable node. This would be a broken feature not a bug fix.